conda/core/index.py

```diff
@@ -27,14 +27,14 @@
 log = getLogger(__name__)
 
 
-def check_whitelist(channel_urls):
-    if context.whitelist_channels:
-        whitelist_channel_urls = tuple(concat(
-            Channel(c).base_urls for c in context.whitelist_channels
+def check_allowlist(channel_urls):
```

Comment:
I think mamba imports this function in their utils.py module:

https://github.com/mamba-org/mamba/blob/da44aadfaf88dfa3ce21656d671682626379cea6/mamba/mamba/utils.py#L71

We can add an alias for one release, with the corresponding deprecation warning?

Reply:
Oh good point

Reply:
Added